pubnub/lib/src/core/crypto/cipher_key.dart

```diff
@@ -21,4 +21,15 @@ class CipherKey {
   factory CipherKey.fromList(List<int> key) {
     return CipherKey._(key);
   }
+
+  @override
+  bool operator ==(dynamic other) {
```

Comment:
@jguz-pubnub  for comparison and to distinguish explicitly provided keyset/using keyset v/s default. Till the time we support cipherKey at configuration level.